### ISSUE TITLE
Allow to pass BasePermission subclasses into `condition` statement

### ISSUE THREAD

**Issue by bogdandm** [rsinger86/drf-access-policy#3]
There is few default classes like `IsAuthorized` but there is no reason to create custom method:
```
def is_authorized(self, *args, **kwargs):
    return IsAuthorized.has_permission(*args, **kwargs)
```

Istead just add check of condition value (`isclass(cond) and issubclass(cond, BasePermission) or isinstance(cond, BasePermission)`) so it can be pass directly:
```
"condition": permissions.IsAuthorized
```

**Comment by bogdandm** [comment]
#5

**Comment by rsinger86** [comment]
Yeah, good point. I had some ideas to handle common condition check, or "built-in conditions", so you don't have to keep writing the same method on every class. I put them in this issue:
https://github.com/rsinger86/drf-access-policy/issues/1

So maybe there could be another built-in condition that looks like this?
```python
{
   "action": "*",
   "principal": "*",
   "condition": { "is_authenticated": true }  # Whether the current user is authenticated
}
```
Though at this point I'm not sure which approach is better.
